### PR TITLE
[New Components] Indicator and Callout

### PR DESCRIPTION
**Pull request by daigof**
- New components: Callout and Indicator
- Review link https://curology-radiance-pr-202.herokuapp.com/
- update prettier rules based on our current standards
- add `md` as auto import extension

### CODE REVIEW

**Comment by michaeljaltamirano** [comment]
Were the dependency changes required for this PR? I don’t think we should be sneaking in major version dependency changes into PRs where that isn’t the focus.

**Comment by daigof** [comment]
> Were the dependency changes required for this PR? I don’t think we should be sneaking in major version dependency changes into PRs where that isn’t the focus.

Was trying to get rid of no single export eslint rule. But just  reverted back those bumps, no really needed

**Comment by daigof** [comment]
@benkolde they are at `8px` did you mean some other value?

File: src/shared-components/callout/style.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { COLORS, SPACER, TYPOGRAPHY_CONSTANTS } from '../../constants';
+
+export const ParentContainer = styled.div`
+  max-width: 327px;
+`;
+
+export const CalloutContainer = styled.div`
+  background-color: ${COLORS.infoBackground};
+  padding: ${SPACER.medium};
+  border-radius: 8px;
```
Comment (daigof) [inline src/shared-components/callout/style.js]: here it sits at `8px`

Comment (benkolde) [reply src/shared-components/callout/style.js]: Oh oops, sorry. It just looked big. I'm gonna approve

**Comment by ida-strom** [comment]
for the callout—the icon should always be 48x48px. would be great to add a note to that and change the demos in heroku so the icons are the same size @daigof

**Pull request merged**